actions/clean-workspace/action.yml

```diff
@@ -9,7 +15,10 @@ runs:
         rm -f config/keystore.jks
         rm -f config/service-account.json
       shell: bash
+
     - name: Stop all emulators
       run: |
-        ./mobile-android-pipelines/scripts/stopAllEmulators.sh
+        ${PIPELINES_DIRECTORY}/scripts/stopAllEmulators.sh
```

Comment:
I haven't seen a project use raw emulators, they all use gradle managed devices, which do not need to be shut down independently, so the stopAllEmulators.sh script should probably be deleted

Reply:
will keep in case a project requires it